docs/DeveloperGuide.adoc

```diff
@@ -508,6 +513,19 @@ The result is then displayed into the GUI for the user as the cost price of the
 [NOTE]
 It is possible for the LoanBook to have a loan that lasts for 0 minutes. This is because there may be some weird edge case where an object is loaned for less than a minute, which gets prorated down.
 
+[big red]#Sequence diagram#
```

Comment:
This part appears as normal text in the dev guide.

Reply:
I have removed it.